config.yaml

```diff
@@ -19,3 +19,20 @@ casa:
     url: "https://biztera.com/api/v1/"
     lookup_period_in_days: 30
     bot_email: 'bizbot@biztera.com'
+foxsec:
+    assignee:
```

Comment:
Should we make this "assignees" rather than a static assignment to a single individual?  We could have 1 person, but this precludes us from having many if we so choose.

Reply:
Happy to make the change to `assignees` to be consistent, however the code for this does not support picking from a list (using pickle, like the other list in the config file). I tried to keep it simple.